spec/yoti/util/age_processor_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe 'Yoti::AgeProcessor' do
+    describe '.has_age_condition' do
+        context 'when there is age_over|under attribute present' do
+            it 'should return true as value' do
+                expect(Yoti::AgeProcessor.has_age_condition('age_over:18')).to eql(true)
+            end
+        end
+
+        context 'when there is no age_over|under attribute present' do
+            it 'should return false as value' do
+                expect(Yoti::AgeProcessor.has_age_condition('age_test:18')).to eql(false)
```

Comment:
'when there is no age_over|under attribute present'
This doesn't seem to be looking for either age_over or age_under.  So either could be present, and the test would still pass.  Yes?

Reply:
This is to make sure it doesn't return `true` when it should return `false`. So it's not looking for neither `age_over` or `age_under`.

Reply:
So, if this test is comparing the name of some attribute against _age_over_ and _age_under_, then maybe we could call it something like:
"returns true when field name is age_over"
and 
"returns false when field name is not age_over or age_under"